java-pubsub-authenticated-push/src/main/webapp/index.jsp

```diff
@@ -0,0 +1,50 @@
+<%@ page import="java.util.List" %>
+<%@ page import="java.util.ArrayList" %>
+<%@ page import="com.google.api.client.googleapis.auth.oauth2.GoogleIdToken" %>
+<%@ page import="io.jsonwebtoken.Claims" %>
+<!doctype html>
+<html>
+<head>
+  <title</title>
```

Comment:
please complete title tag and add proper title

Reply:
done